_opcodes/op152-bgee.html

```diff
@@ -0,0 +1,75 @@
+---
+n: 152
+name: "Spell Effect: Play Movie"
+param1: "Irrelevant"
+param2: "Irrelevant"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Plays the movie specified by the resource key.
+<br/>Known movies are:
+<ul>
+	<li>Common:
+		<ul>
+			<li>LOGO – WotC logo</li>
+		</ul>
+	</li>
+
+	<li>BG1:EE
+		<ul>
+			<li>DEATHAND – Death</li>
+			<li>INTRO – Baldur's Gate: Enhanced Edition Introduction</li>
+			<li>REST – Resting Outside</li>
+			<li>RESTDUN – Resting in Dungeon</li>
+			<li>RESTINN – Resting at Inn</li>
+			<li>BGENTER – Entering Baldur's Gate</li>
+			<li>FRARMINN – Friendly Arm Inn</li>
+			<li>NASHKELL – Nashkell</li>
+			<li>CAMP – Bandit Camp</li>
+			<li>MINEFLOD – Flooding Cloakwood Mines</li>
+			<li>ELDRCITY – Undercity</li>
+			<li>ENDMOVIE – Epilogue</li>
+			<li>ENDCRDIT – Credits</li>
+			<li>WRECK – Wrecked Ship (Werewolf Island)</li>
+			<li>BLACKPIT – The Black Pits Introduction</li>
+		</ul>
+	</li>
+
+	<li>BG:SoD
+		<ul>
+			<li>SODCIN01 – Siege of Dragonspear Introduction</li>
+			<li>SODCIN02 – Boarskyr Bridge</li>
+			<li>SODCIN03 – Murder</li>
+			<li>SODCIN05 – Epilogue</li>
+		</ul>
+	</li>
+
+	<li>BG2:EE
+		<ul>
+			<li>NITEDAY – Sunrise</li>
+			<li>DAYNITE – Sunset</li>
+			<li>DEATHAND – Death</li>
+			<li>INTRO15F – Shadows of Amn Intro</li>
+			<li>FLYTHR01 – Asylum</li>
+			<li>FLYTHR02 – Underdark</li>
+			<li>FLYTHR03 – Suldanessellar</li>
+			<li>FLYTHR04 – Hell</li>
+			<li>INTRO – Throne of Bhaal Intro</li>
+			<li>POCKETZZ – Pocket Plane Rest</li>
+			<li>SARADUSH – Saradush Flythrough</li>
+			<li>MANAFORG – Mana Forge Flythrough</li>
+			<li>MELISSAN – Melissan's Demise</li>
+			<li>OUTRO – Closing the Book</li>
+			<li>HEXXATDR – Dragomyr's Crypt (Hexxat)</li>
+			<li>DORNGORG – Resurrection Gorge (Dorn)</li>
+			<li>DORNGORG – Ritual Circle (Dorn)</li>
+			<li>NEERAES – Town Bell (Neera)</li>
+			<li>RASAADMI – Sharran (Rasaad)</li>
+			<li>RASAADTE – Temple (Rasaad)</li>
+		</ul>
+	</li>
+</ul>
```

Comment:
I suggest you add this to the bg2 one and convert the lists to a table, one column per game. Will keep everything shorter.

Reply:
Done.